web/angular-wallet/src/app/app.component.ts

```diff
@@ -150,13 +153,22 @@ export class AppComponent extends UnsubscribeOnDestroy implements OnInit, OnDest
     try {
       const blockchainStatus = await this.networkService.getBlockchainStatus();
       this.isScanning = !this.firstTime && (this.previousLastBlock !== blockchainStatus.lastBlock);
+      this.numberOfBlocks = blockchainStatus.numberOfBlocks;
+      this.lastBlockchainFeederHeight = blockchainStatus.lastBlockchainFeederHeight;
+      this.percentDownloaded = Math.round(blockchainStatus.numberOfBlocks / blockchainStatus.lastBlockchainFeederHeight * 100);
+      this.downloadingBlockchain = this.percentDownloaded < 100;
       this.previousLastBlock = blockchainStatus.lastBlock;
       if (this.isScanning) {
-        await this.updateAccountsAndCheckBlockchainStatus(blockchainStatus);
+        setTimeout(async () => {
+          await this.updateAccountsAndCheckBlockchainStatus(blockchainStatus);
+        }, 1000);
```

Comment:
This was absolutely spamming the API from time to time.

Reply:
ok. I would use a constant, as it reused a few lines below.... but that's really _mimimi_ :stuck_out_tongue: